index.js

```diff
@@ -274,11 +273,10 @@ Kareem.prototype.wrap = function(name, fn, context, args, options) {
         options, lastArg);
     }
 
-    const end = (typeof lastArg === 'function' ? args.length - 1 : args.length);
     const numParameters = fn.length;
     let ret;
     try {
-      ret = fn.apply(context, args.slice(0, end).concat(_cb));
+      ret = fn.apply(context, argsWithoutCb.concat(_cb));
```

Comment:
we could actually do a argsWithoutCb.push(_cb) instead of an concat, which should be even faster
and then in line 295 we dont need to do + 1, but then the name "argsWithoutCb" should be changed to something without indicating that there is no callback...

Reply:
I think this is neglectable